.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: build and publish
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  rspec_before_build:
+    if: github.event.base_ref == 'refs/heads/master'
+    uses: ./.github/workflows/_rspec.yml
+  release:
+    if: github.event.base_ref == 'refs/heads/master'
+    needs: rspec_before_build
+    runs-on: ubuntu-latest
+    steps:
+      - name: setup ruby
+        uses: ruby/setup-ruby@v1.100.0
+        # we do not need to set ruby version since there is a .ruby-version file
+      - name: checkout
+        uses: actions/checkout@v2
+      - run: bundle install
+      - name: Publish to RubyGems
+        run: |
+          mkdir -p $HOME/.gem
+          touch $HOME/.gem/credentials
+          chmod 0600 $HOME/.gem/credentials
+          printf -- "---\n:rubygems_api_key: ${GEM_HOST_API_KEY}\n" > $HOME/.gem/credentials
+          # Be explicit with the gem name
+          gem build bundler-fixture.gemspec
+          # Be explicit with the version. Will fail if Bundler::Fixture::VERSION is not the same.
+          gem push bundler-fixture-${TAG_NAME##v}.gem
+        env:
+          TAG_NAME: "${{github.ref_name}}"
+          GEM_HOST_API_KEY: "${{secrets.RUBYGEMS_AUTH_TOKEN}}"
```

Comment:
I made a few assumptions by adding this. First is, there will be a secret `RUBYGEMS_AUTH_TOKEN` for this repository for pushing to rubygems. Second is about the workflow itself. This workflow will be triggered by pushing a version tag to master. This requires `Bundler::Fixture::VERSION` to be the same as the tag. Without the leading `v` of course.

Reply:
Hi @chrismo ! If you have a minute or two could you please verify that there is the `RUBYGEMS_AUTH_TOKEN` secret for this repository or add it if not? I cannot see or edit secrets for this repository, which is fine BTW. Also I cannot push to rubygems with my token to your namespace. Thanks.